demo/TG.py

```diff
@@ -134,6 +134,7 @@ def regression_test(context):
                      mesh="triplyperiodic")
 
     context = sol.get_context()
+    context['K'] = [array(_) for _ in context['K']]
```

Comment:
Don't much like this one. First, it can be done nicer, with `context.K = np.array(context.K)`. Second, it should not be necessary. `context.K` is intentionally very sparsely stored using nice numpy broadcasting rules. You have 

- `context.K[0].shape` = (32, 32, 17)
- `context.K[0].strides` = (8, 0, 0)

and same for `K[1]` and `K[2]`. So even though `context.K[i]` looks and acts like a 3D array of shape (32, 32, 17), it is stored with 32 numbers only and stride magic. Doing `context.K = np.array(context.K)` creates a dense array of shape (3, 32, 32, 17) and strides (139264, 4352, 136, 8).

So, what I'm saying is that this should preferably be taken care of in the `pythran_maths.py` module. It is taken care of for cython in `cython_maths.in` `_cross3`. Any way to do this for pythran?

Reply:
yep, I'll do that!